docs/examples/spatial_transformation/gaussian_pyramid.jl

```diff
@@ -0,0 +1,28 @@
+# ---
+# cover: assets/lake_color.tif
+# title: Gaussian Pyramid
+# author: Park ma
+# date: 2021-5-24
+# ---
+
+# This demo illustrates the use of [Gaussian Pyramid](https://juliaimages.org/stable/function_reference/#Images.gaussian_pyramid)
+
+# Gaussian Pyramid is a technique in image processing that breaks down 
+# an image into successively smaller groups of pixels to blur it. 
+
+using TestImages,Images
+
+img = testimage("lake_color")
+
+# We will build image pyramids though the Gaussian Pyramid.
+
+img_pyramids = gaussian_pyramid(img,5,2,2.0)
```

Comment:
For self completeness, I feel that we need to explain each input of `guassian_pyramid` here, and the general idea of how to choose the parameters.